src/main/scala/it/pagopa/interop/partymanagement/model/party/PersistedPartyRelationship.scala

```diff
@@ -54,7 +54,7 @@ final case class PersistedPartyRelationship(
     id = id,
     to = to,
     product = product.toRelationshipProduct.id,
-    digitalAddress = institutionUpdate.map(_.toInstitutionUpdate.digitalAddress).getOrElse("")
+    digitalAddress = institutionUpdate.map(_.toInstitutionUpdate.digitalAddress.getOrElse("")).get
```

Comment:
Calling `.get` on a `None` will throw an Exception.

How about changing the method signature to one of
```scala
def toInstitutionId: Option[InstitutionId]
def toInstitutionId: Try[InstitutionId]
def toInstitutionId: Either[Throwable,InstitutionId]
```
and handle the failure at call site?

Reply:
Why not a simple?
`institutionUpdate.flatMap(_.toInstitutionUpdate.digitalAddress).getOrElse("")`

Reply:
I like this suggestion